apps/game/client/functions.ts

```diff
@@ -33,27 +33,17 @@ export const newPhoneProp = async () => {
     const [x, y, z] = GetEntityCoords(playerPed, true);
     global.phoneProp = CreateObject(GetHashKey(phoneModel), x, y, z + 0.2, true, true, true);
     const boneIndex = GetPedBoneIndex(playerPed, 28422);
-    AttachEntityToEntity(
```

Comment:
This shouldn't be collapsed, eslint should warn/error about this

Reply:
no warning or errors unless im just missing something

Reply:
i made the change anyways, not sure what difference it actually makes